pkgs/applications/networking/instant-messengers/rambox/bare.nix

```diff
@@ -3,13 +3,13 @@
 
 stdenv.mkDerivation rec {
   name = "rambox-bare-${version}";
-  version = "0.6.1";
+  version = "0.6.2";
 
   src = fetchFromGitHub {
     owner = "saenzramiro";
```

Comment:
It would be proper to fetch from the new repo `ramboxapp/community-edition` since https://github.com/saenzramiro/rambox redirects to it anyway.

```shell
$ nix-prefetch-url --type sha256 --unpack https://github.com/ramboxapp/community-edition/archive/0.6.2.tar.gz
unpacking...
path is '/nix/store/xzal00d5qk87wc9cn14kwpn86sqc1mpp-0.6.2.tar.gz'
150vf62cp739l9dgpnksgpkffabs2wi15q217m3nai34irhwzk8m
```

Reply:
fixed